### PR TITLE
feat: SVM dashboard with NFSv3 Frontend Drilldown

### PR DESCRIPTION
**Pull request by fvsqr**
Dashboard for SVM, for now with drilldown for NFSv3 only. 

scrnsht 2021-07-21 um 17 34 27

### CODE REVIEW

**Comment by fvsqr** [comment]
@cla-bot check

**Comment by fvsqr** [comment]
@cla-bot check

**Comment by rahulguptajss** [comment]
@fvsqr thanks for your PR. You would need to sign ccla as mentioned below before we can review it.

> Thank you for your interest in contributing to the Harvest project! We require contributors to sign our [Corporate contributor license agreement (CCLA)](https://github.com/NetApp/harvest/blob/main/CONTRIBUTING.md#creating-a-pull-request), and we don"t have the user(s) @burkl on file. In order for us to review and merge your code, please follow the instructions in step 6 of [creating a pull request](https://github.com/NetApp/harvest/blob/main/CONTRIBUTING.md#creating-a-pull-request).
> After signing the CCLA, you can ask us to recheck this PR by posting `@cla-bot check` as a comment to the PR.

**Comment by burkl** [comment]
@cla-bot check

**Comment by cgrinds** [comment]
Hi @burkl I received your signed CCLA, let me update the bot

File: grafana/prometheus/harvest_dashboard_svm.json [hunk 1]
```diff
@@ -0,0 +1,1174 @@
...
+          "placement": "right"
+        },
+        "tooltip": {
+          "mode": "single"
+        }
+      },
+      "pluginVersion": "8.0.6",
+      "targets": [
+        {
+          "exemplar": true,
```
Comment (rahulguptajss) [inline grafana/prometheus/harvest_dashboard_svm.json:751]: exemplar needs to be false. We had issues with it in past.  More information at https://github.com/NetApp/harvest/issues/96

**Review by cgrinds — APPROVED** [review]
Thanks @burkl appreciate the contribution! there are a few changes we'll make around `exemplar` but this looks like a great start to filling a gap with Harvest 1.6

**Pull request merged**